Gemfile

```diff
@@ -1,9 +1,20 @@
+# frozen_string_literal: true
+
 source "https://rubygems.org"
 
 # Specify your gem's dependencies in bibliothecary.gemspec
 gemspec
 
+group :development do
+  gem "pry"
+  gem "rake", "~> 12.0"
+  gem "rspec", "~> 3.0"
+  gem "rubocop", "~> 1.71"
+  gem "rubocop-rails"
+  gem "webmock"
+end
```

Comment:
TIL that we should use `add_dependency` in the gemspec, avoid `add_runtime_dependency`/`add_development_dependency`, and just put dev dependencies n Gemfile.